VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts

```diff
@@ -28,22 +30,28 @@ export class CommandCentreComponent implements OnInit, OnDestroy {
     public menuOption = MenuOption;
 
     private readonly judgeAllocationStorage: SessionStorage<string[]>;
+    private readonly courtAccountsAllocationStorage: SessionStorage<CourtRoomsAccounts[]>;
 
     venueAllocations: string[] = [];
+    courtRoomsAccountsFilters: CourtRoomsAccounts[] = [];
 
     selectedMenu: MenuOption;
 
     conferencesSubscription: Subscription;
     eventHubSubscriptions: Subscription = new Subscription();
+    filterSubcription: Subscription;
 
     hearings: HearingSummary[];
     selectedHearing: Hearing;
+    cloneHearings: HearingSummary[] = [];
```

Comment:
The clone hearings is the original, not filtered data, if need to return original set after Cancel button pressed, it will return this clone instead of reading from database.

Reply:
Could you please rename this variable so it is more clear for anyone who works on this in the future please. Thanks

Reply:
Please advice the name for variable?

Reply:
originalHearings

Reply:
done